packages/react-form-hooks/src/hooks/errorPropagation.ts

```diff
@@ -0,0 +1,28 @@
+import {useEffect} from 'react';
+import get from 'get-value';
```

Comment:
This tiny?

Reply:
V tiny
![image](https://user-images.githubusercontent.com/4889856/56371076-6eb2a400-61ca-11e9-8928-422d243e217d.png)